cpp/ql/lib/change-notes/2024-06-20-extensible-allocation-deallocation.md

```diff
@@ -0,0 +1,4 @@
+---
+category: feature
+---
+* It is now possible to extend the classes `AllocationFunction` and `DeallocationFunction` via data extensions. Extensions of these classes should be added to `lib/ext/allocation` and `lib/ext/deallocation` respectively.
```

Comment:
```suggestion
* It is now possible to extend the classes `AllocationFunction` and `DeallocationFunction` via data extensions. Extensions of these classes should be added to the `lib/ext/allocation` and `lib/ext/deallocation` directories respectively.
```
to make things a little clearer, as we have more than one thing of the form `a/b/c` going on in this area.

Reply:
Good idea!